packages/desktop-client/src/hooks/useAccountPreviewTransactions.ts

```diff
@@ -83,10 +84,10 @@ export function useAccountPreviewTransactions({
   return useMemo(() => {
     if (!accountId) {
       return {
-        previewTransactions: allPreviewTransactions,
-        runningBalances: allRunningBalances,
-        isLoading,
-        error,
+        previewTransactions: [],
+        runningBalances: new Map(),
+        isLoading: false,
+        error: undefined,
```

Comment:
This reverts the fix in https://github.com/actualbudget/actual/pull/5135, is that intended?